deploy/cr.yaml

```diff
@@ -1,4 +1,8 @@
 apiVersion: "androidsdk.aerogear.org/v1"
 kind: "AndroidSDK"
 metadata:
-  name: "example"
+  name: "android-sdk-pkg-config"
+spec:
+  ConfigMapName: android-sdk-config
+status: 
+  Phase: Install
```

Comment:
The Phase should probably be empty. My understanding is that the status is controlled by the operator not the user

Reply:
:+1: